### PR TITLE
Adding Container Instance for 12.5.0-beta release.

### PR DESCRIPTION
**Pull request by marstr**
Adding Container Instance API Versions, and bumping the Minor Version number.

---

Thanks you for your contribution to the Azure-SDK-for-Go! We will triage and review it as quickly as we 
can.

As part of your submission, please make sure that you can make the following assertions:

 - [ ] I'm not making changes to Auto-Generated files which will just get erased next time there's a release.
   - If that's what you want to do, consider making a contribution here: https://github.com/Azure/autorest.go
 - [ ] I've tested my changes, adding unit tests where applicable.
 - [ ] I've added Apache 2.0 Headers to the top of any new source files.
 - [ ] I'm submitting this PR to the `dev` branch, or I'm fixing a bug that warrants its own release and I'm targeting the `master` branch.
 - [ ] If I'm targeting the `master` branch, I've also added a note to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-go/blob/master/README.md).
 - [ ] I've mentioned any relevant open issues in this PR, making clear the context for the contribution.

### CODE REVIEW

**Comment by mcardosos** [comment]
@marstr There are breaking changes in this PR, it should not be an additive change.

**Comment by marstr** [comment]
Woah, I don't see them. Where are the breaking changes, @mcardosos?

File: services/containerinstance/mgmt/2017-10-01-preview/containerinstance/models.go [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+// ContainerPropertiesInstanceView the instance view of the container instance. Only valid in response.
+type ContainerPropertiesInstanceView struct {
+	// RestartCount - The number of times that the container instance has been restarted.
+	RestartCount *int32 `json:"restartCount,omitempty"`
+	// CurrentState - Current container instance state.
+	CurrentState *ContainerState `json:"currentState,omitempty"`
+	// PreviousState - Previous container instance state.
+	PreviousState *ContainerState `json:"previousState,omitempty"`
+	// Events - The events of the container instance.
+	Events *[]Event `json:"events,omitempty"`
```
Comment (mcardosos) [inline services/containerinstance/mgmt/2017-10-01-preview/containerinstance/models.go]: Hello breaking change

Comment (marstr) [reply services/containerinstance/mgmt/2017-10-01-preview/containerinstance/models.go]: This is in a completely new API Version, it's not a breaking change. :)

Comment (mcardosos) [reply services/containerinstance/mgmt/2017-10-01-preview/containerinstance/models.go]: Brainfart

**Comment by marstr** [comment]
Alright folks, as soon as CI passes, I believe this is ready to go. Mind taking another look, @jhendrixMSFT, @vladbarosan, @mcardosos?

**Comment by mcardosos** [comment]
Could we add that some services were added to the profiles in the changelog?

**Pull request merged**